VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public const string Password = "abc";
```

Comment:
**CODE_SMELL**
> Change this constant to a 'static' read-only property.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU9&pullRequest=16&resolved=false)

Reply:
/falsepositive

Reply:
/falsepositve

Reply:
/falsepositive

Reply:
The issue is already resolved or confirmed.

Reply:
/falsepositive

Reply:
/falsepositive

Reply:
The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU9&pullRequest=16).

Reply:
The issue is already resolved or confirmed.